README.md

```diff
@@ -448,11 +448,10 @@ Then use the "Launch new AppServer" button in the web UI to provision a server
 with the updated settings, and click "Activate this app server" to use the new
 server when it's ready.
 
-**To delete an instance**, ensuring that all virtual machines are terminated, run:
+**To delete an instance**, which will also terminate all associated AppServers
+and their virtual machines, run:
 
 ```python
-for appserver in instance.appserver_set.all():
-    appserver.terminate_vm()
```

Comment:
Edit: added in cba5216d2ccce9d3c4da970eb3f4949d224feeb0 .

I'm going to rebase and squash now because GitHub won't let me merge as-is.